x-pack/plugins/infra/public/components/logging/log_entry_examples/log_entry_examples.tsx

```diff
@@ -45,5 +45,4 @@ const Wrapper = euiStyled.div`
   align-items: stretch;
   flex-direction: column;
   flex: 1 0 0%;
-  overflow: hidden;
```

Comment:
This removal seems to break the rendering on the categories tab:

Before:
![image](https://user-images.githubusercontent.com/973741/99980325-678d4800-2da8-11eb-89f2-f184181a9991.png)

After:
![image](https://user-images.githubusercontent.com/973741/99980377-783dbe00-2da8-11eb-9682-c9b776b4e0df.png)

I assume the intention for this change was to prevent the button from being cut off? Could some margin/padding around that button or the "empty state" component do this instead?

Reply:
I cannot seem to reproduce in either Chrome (above) or Firefox (below). Is it possible your system had some stylesheet cache that didn't bust when it should? Do you mind trying again after a `yarn kbn clean && yarn kbn bootstrap`?

<img width="1680" alt="Screenshot 2020-11-23 at 21 26 03" src="https://user-images.githubusercontent.com/57448/100011937-c23b9980-2dd2-11eb-92ae-d57e3718af86.png">

Reply:
I can still reliably produce it after `kbn clean` on Chromium 87.0.4280.66 and Firefox 83.0 if I reduce the window width below 2040 px.

Reply:
I see. It happens when resizing the window. I'll take a look

Reply:
Fixed!